apps/website/blog/2024-10-29-q3-review.md

```diff
@@ -0,0 +1,61 @@
+
+As always, if you're interested in running a round for your community, please get in [touch](https://discord.gg/akDcCB69)
+
+## Coordinator Service ✅
+
+The Coordinator Service is the ultimate automation of MACI’s main pain-points. It now includes three new features for the frontend interface to provide a very neat experience to MACI operators.
+
+Because we prioritise robust security features whenever possible, the service does not hold any wallet seed phrase to communicate with the various chains. Instead, we implemented a flow that uses ephemeral session keys for short lived approvals, greatly decreasing the impact of a key compromise. This is all possible thanks to account abstraction features such as session keys.
+
+With the flow described above, we implemented a deployer service to easily deploy all needed MACI contracts, as well as a an endpoint to merge the MACI state and message trees, an important step required to be able to tally the results.
+
+## Account Abstraction Exploration ✅
+
+Together with a fellow PSE team developer, [John](https://github.com/JohnGuilding), the MACI team is working on a [fork of the MACI platform](https://github.com/privacy-scaling-explorations/hacking-pse) that implements account abstraction features, such as smart wallets and paymasters.
+
+[This fork](https://github.com/privacy-scaling-explorations/hacking-pse) was used in PSE’s first internal hackathon, and allows PSE and EF team members to vote on their favourite hackathon project without spending a single cent from their wallet onchain.
+
+This was a successful experiment which will prove fundamental in our future efforts to improve the MACI platform and make it even more accessible to a general “non crypto native” user base.
+
+Stay tuned for a case study of this experiment!
+
+## MACI Protocol Improvements ✅
+
+The [3327 team](https://3327.io) successfully completed their [Anonymous Poll Joining grant](https://maci.pse.dev/blog/anonymous-poll-joining). This work enabled unconditional voter privacy, further strengthening user privacy in the protocol. For more details, please refer to their [blog post](https://maci.pse.dev/blog/anonymous-poll-joining).
+
+With regards to implementing Vitalik’s latest MACI proposal [Mostly Offchain Happy Path](https://ethresear.ch/t/maci-with-mostly-off-chain-happy-path/19527/), we have not been able to start due to other priorities, however we have defined the tasks required and unblocked the epic by implementing the session keys flow on the coordinator service, so that it can be re-used for a secure relayer service.
```

Comment:
```suggestion
With regard to implementing Vitalik’s latest MACI proposal [Mostly Offchain Happy Path](https://ethresear.ch/t/maci-with-mostly-off-chain-happy-path/19527/), we have not been able to start due to other priorities, however we have defined the tasks required and unblocked the epic by implementing the session keys flow on the coordinator service, so that it can be re-used for a secure relayer service.
```